string mod_selection
---
bool response